README.md

```diff
@@ -4,6 +4,14 @@ The Terraform MCP Server is a [Model Context Protocol (MCP)](https://modelcontex
 server that provides seamless integration with Terraform Registry APIs, enabling advanced
 automation and interaction capabilities for Infrastructure as Code (IaC) development.
 
+## Features
```

Comment:
Nit pick - Use case and features talk about the same thing, let's maybe remove use cases or features

I think removing use case is okay

Reply:
resolved in the new commit